lib/rules/jsdoc.js

```diff
@@ -22,8 +22,8 @@
  *
  * There are 3 presets for `Closure Compiler`, `JSDoc3` and `JSDuck5`.
  *
- * By default it allows any tag of mixed set. You can pass `Object`
- * to select preset with `preset` field and add custom tags with `extra` field.
+ * By default it allows any tag of mixed set. You can pass `Object` to select
+ * preset with `preset` field and add custom tags with `extra` field.
```

Comment:
Is there another word for `tag` or is that the correct jsdoc terminology? Maybe `By default, it allows for any combination of tags?`

`to select preset` -> `to select a preset`?